annual-survey.md

```diff
@@ -0,0 +1,38 @@
+# Rust annual survey guide
+This document describes a brief guide on how to prepare the yearly Rust annual survey. It is probably not complete, but
+it should nevertheless be helpful as a guide.
+
+1. Copy survey from last year and perform basic modifications
+   - Create `surveys/<year>-annual-survey/questions.md` and copy questions from the last year into it
+   - Change basic time information, e.g. bump years
+   - Change Rust versions (usually they move forward by ~10 versions, based on the 6-week schedule)
+   - See [#234](https://github.com/rust-lang/surveys/pull/234) for an example PR
+2. Go through the [issue tracker](https://github.com/rust-lang/surveys/issues) and try to resolve issues from the last year
+   - Try to find issues that could be applicable, and modify the questions accordingly via PRs to this repo
+3. Ask for general feedback for question/answer additions/modifications
+   - Ideally, ask in the [#rust-survey](https://rust-lang.zulipchat.com/#narrow/stream/402479-t-community.2Frust-survey) or
+   [#general](https://rust-lang.zulipchat.com/#narrow/stream/122651-general) Zulip streams, and also try to ask the individual
+   top level teams (`t-compiler`, `t-lang`, `t-cargo` etc.).
+4. Finalize the questions/answers for the given year (ideally by **end of September**)
+5. Put the survey questions to SurveyHero
+   - This needs collaboration with the Rust foundation, please ask in the [#rust-survey](https://rust-lang.zulipchat.com/#narrow/stream/402479-t-community.2Frust-survey)
+   Zulip stream for assistance.
+6. Find translator volunteers
+   - Either for translating the survey, or for reviewing auto-generated translations.
+   - You can ask around on the `#rust-survey` or `#general` Zulip streams for assistance.
```

Comment:
I'd expand a bit this section to mention the hidden traps in the translation workflow using SH (feel free to improve the wording)

```suggestion
6. Translate the questionnaire
   - SurveyHero can generate the questionnaire in other languages, uses some machine translation which often leaves a bit to desire
   - Find translators for reviewing the survey and the auto-generated translations
   - You can ask around on the `#rust-survey` or `#general` Zulip streams for assistance.
   - Be aware that the SurveyHero export feature is currently broken (breaks non Latin alphabets) and importing the translations is not allowed. Correcting the translations is therefore a manual work both for the translator and for the operator that has access to the SurveyHero backoffice.
   - Translators can take advantage of [this small guide](https://rust-lang.zulipchat.com/#narrow/stream/402479-t-community.2Frust-survey/topic/Translation.20guide/near/406836813)
```